### PR TITLE
Add capitalone.com password rules

### PR DESCRIPTION
**Pull request by grantwiggins**

### CODE REVIEW

**Comment by rmondello** [comment]
Hi @grantwiggins! Can you provide the information from capitalone.com’s website that you used to derive this rule?

File: quirks/password-rules.json [hunk 1]
```diff
@@ -35,6 +35,9 @@
     "brighthorizons.com": {
         "password-rules": "minlength: 8; maxlength: 16;"
     },
+    "capitalone.com": {
+        "password-rules": "minlength: 8; maxlength: 32; required: lower, upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_./\@$*&!#];"
```
Comment (rmondello) [inline quirks/password-rules.json:39]: Because lower, upper, and digit are required, you don’t need to specify them in allowed.

**Comment by grantwiggins** [comment]
> Hi @grantwiggins! Can you provide the information from capitalone.com’s website that you used to derive this rule?

Annotation 2020-06-07 144854

File: quirks/password-rules.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
         "password-rules": "minlength: 8; maxlength: 16;"
     },
     "capitalone.com": {
-        "password-rules": "minlength: 8; maxlength: 32; required: lower, upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_./\@$*&!#];"
+        "password-rules": "minlength: 8; maxlength: 32; required: lower, upper; required: digit; allowed: [-_./\@$*&!#];"
```
Comment (DerJuulsn) [inline quirks/password-rules.json:39]: \ doesnt work in JSON use \\\

File: quirks/password-rules.json [hunk 3]
```diff
@@ -35,6 +35,9 @@
     "brighthorizons.com": {
         "password-rules": "minlength: 8; maxlength: 16;"
     },
+    "capitalone.com": {
+        "password-rules": "minlength: 8; maxlength: 32; required: lower, upper; required: digit; allowed: [-_./\\@$*&!#];"
```
Comment (rmondello) [inline quirks/password-rules.json:39]: I’m sad that you had to escape this. Maybe we should explore data formats that don’t involve having to escape the password rules string.

**Pull request merged**